examples/custom-service.mdx

```diff
@@ -20,49 +38,117 @@ edition = "2021"
 [lib]
 
 [dependencies]
-shuttle-service = { version = "0.9.0" }
-shuttle-shared-db = { version = "0.9.0", features = ["postgres"] }
-sqlx = { version = "0.6", features = ["runtime-tokio-native-tls", "postgres"] }
+anyhow = "1.0.62"
+axum = "0.6.4"
+hyper = "0.14.24"
+poise = "0.5.2"
+serde = "1.0"
+shuttle-secrets = "0.9.0"
+shuttle-service = { version = "0.9.0", features = ["bot-serenity"] }
```

Comment:
Hmm, do we need any features on shuttle-service when we implement service ourselves?